tests/console/yast2_nfs4_client.pm

```diff
@@ -101,8 +106,8 @@ sub run {
 
     # Test NFSv4 ACL
     assert_script_run "nfs4_getfacl /tmp/nfs/client/secret.txt";
-    assert_script_run "nfs4_setfacl -R -a A:df:$testapi::username\@localdomain:RX /tmp/nfs/client/secret.txt";
-    assert_script_run "nfs4_getfacl /tmp/nfs/client/secret.txt | grep \"A::`id -u $testapi::username`:rxtcy\"";
+    script_run("nfs4_setfacl -R -a A:df:$testapi::username\@localdomain:RX /tmp/nfs/client/secret.txt") && record_soft_failure("bsc#1157915, Set recursive options for file");
```

Comment:
Not great. We are trying to give recursive ACL attributes to a file, which is illogic, recursive is for directories. I thought first of removing those recursive arguments (-R and :df:), but this is somehow a bug, normally setfacl now "removes" the directory attributes if we run it against a file, see this comment: https://bugzilla.suse.com/show_bug.cgi?id=1157915#c5. 
But then with the workaround, the next line also fails.
I am open to suggestions here.